### PR TITLE
chore: usage of internal docs resources

### PR DESCRIPTION
**Pull request by M0Rf30**
# Description
This enables the usage of preview with the internal docs core component

## Type of change

It accomplish the task in two ways:
- enabling custom PYTHONPATH for uno.py within docs-core
- removing the UserInstallation command-line arg embedded within unoserver/server.py

`/tmp` usage is prone to failures (simply doesn't work).

In this way we leverage the `SYSUSERCONFIG/.config/core/4` (in our case `/var/lib/carbonio-preview/.config/core/4`)

It also sports some cleanup of dependencies and unoserver bump to 1.2 (removal of `daemon` parameter)

### CODE REVIEW

**Review by federicorispo — DISMISSED** [review]

**Review by RakuJa — COMMENTED** [review]
I'll wait to test on rhel8 before reviewing

**Comment by M0Rf30** [comment]
> I'll wait to test on rhel8 before reviewing

Also rhel now is working with https://github.com/Zextras/carbonio-preview-ce/pull/20/commits/885b451c210de0786c583bac54970d5a579291a2

File: package/PKGBUILD [hunk 1]
```diff
@@ -43,16 +35,16 @@ makedepends:yum=(
...
 
 provides:yum=(
-  "libjpeg-62ed1500.so.62.3.0(LIBJPEG_6.2)(64bit)"
+  "libjpeg-3e290cba.so.62.3.0(LIBJPEG_6.2)(64bit)"
   "liblzma-d540a118.so.5.2.5(XZ_5.0)(64bit)"
-  "libpng16-213e245f.so.16.37.0(PNG16_0)(64bit)"
+  "libpng16-52f22300.so.16.37.0(PNG16_0)(64bit)"
   "libz-dd453c56.so.1.2.11(ZLIB_1.2.3.4)(64bit)"
   "libz-dd453c56.so.1.2.11(ZLIB_1.2.9)(64bit)"
-  "/bin/python3"
```
Comment (RakuJa) [inline package/PKGBUILD:55]: On rhel8 there is a problem with the installation. Is this line the culprit?

[root@sw-r01 ~]# yum install ./carbonio-preview-ce-0.2.11.x86_64\(1\).rpm
Updating Subscription Management repositories.
Last metadata expiration check: 1:41:39 ago on lun 24 ott 2022 07:29:11 CEST.
Error:
Problem: conflicting requests
- nothing provides /bin/python3.8 needed by carbonio-preview-ce-0.2.11-1.el8.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)

Comment (M0Rf30) [reply package/PKGBUILD:55]: Seems to happen only on rhel8

**Comment by RakuJa** [comment]
Incremented docs-core timeout. Docs-core needs more time to startup and killing it often with a short timeout result in a loop consisting of startup-shutdown [c6613ac](https://github.com/Zextras/carbonio-preview-ce/pull/20/commits/c6613ac5485bd0910896723bbc3427f8497ce88c)

**Review by federicorispo — APPROVED** [review]
:rocket:

**Review by RakuJa — APPROVED** [review]
LGTM, works on rhel8 and debian!

**Pull request merged**